File: legacy_scripts/check_number.py
import requests

token = 'shJOb5wskQMTyfoF20GLqmJOclA5if5j'
phone = '37257825047'

headers = {
    'Authorization': f'Bearer {token}',
    'Accept': 'application/json'
}

url = f'https://gate.whapi.cloud/contacts/{phone}'
try:
    response = requests.get(url, headers=headers)
    print(f"Status Code: {response.status_code}")
    print(f"Response: {response.json()}")
except Exception as e:
    print(f"Error: {e}")

File: legacy_scripts/check_number.js
const axios = require('axios');

const JUANA_TOKEN = "shJOb5wskQMTyfoF20GLqmJOclA5if5j"; // From Dynamic WhatsApp Alert

async function checkNumber(phone) {
    try {
        const response = await axios.get(`https://gate.whapi.cloud/contacts/${phone}`, {
            headers: { 'Authorization': `Bearer ${JUANA_TOKEN}` }
        });
        console.log(`Number ${phone} check result:`, response.data);
    } catch (e) {
        console.log(`Number ${phone} error:`, e.response?.status, e.response?.data);
    }
}

async function run() {
    await checkNumber('584162013551');
    await checkNumber('584162013551@s.whatsapp.net');
}

run();
